### DISCUSSION TITLE
Как узнать, что строка последняя в колонке?

### DISCUSSION THREAD

**Discussion by UPRLS9000** [totumonline/totum-ru-issues-and-discussions#461] [Как это сделать?] [answered]
Здравствуйте.
Подскажите, пожалуйста, как узнать, что строка последняя в колонке?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Берете селектом id последней строки по order desc который вам нужен и если он совпадает с id текущей строки - эта строка последняя.

**Reply by UPRLS9000** [reply]
Спасибо
